### ISSUE TITLE
How to apply it to Mask R-CNN

### ISSUE THREAD

**Issue by hanb257** [kemaloksuz/RankSortLoss#8]
RS Loss is based on torch, while Mask R-CNN is based on tensorflow, I want to know how to apply it to Mask
R-CNN.

**Comment by kemaloksuz** [comment]
Mask R-CNN can be implemented by different programming languages and different deep learning frameworks (e.g. Pytorch or Tensorflow). In otherwords, using Tensorflow with Mask R-CNN is not binding. Accordingly, in this repository, Mask R-CNN is implemented via Pytorch. You can check out the implementation and use it without any problem.
